packages/pn-pa-webapp/src/api/auth/Auth.api.ts

```diff
@@ -13,6 +13,7 @@ export const AuthApi = {
             fiscal_number: response.data.fiscal_number,
             organization: response.data.organization,
             sessionToken: response.data.sessionToken,
-            uid: response.data.uid
+            uid: response.data.uid,
+            hasGroups: response.data.hasGroups,
```

Comment:
hasGroups should be a field inside organization

Reply:
so, I should have response.date.organization.hasGroup instead?

Reply:
yes, it should